ffs/integrationtest/integration_test.go

```diff
@@ -1275,6 +1275,38 @@ func TestResumeScheduler(t *testing.T) {
 	require.Equal(t, 1, len(sh.Cold.Filecoin.Proposals)) // Check only one deal still exits.
 }
 
+func TestParallelExecution(t *testing.T) {
```

Comment:
Add to tests a case where three Cids are pushed and all of them should be in _Executing_ immediately. An of course, finish correctly.